.github/workflows/code_freeze_start.yml

```diff
@@ -18,6 +18,18 @@ jobs:
       - name: Clone dd-trace-dotnet repository
         uses: actions/checkout@v3
 
+      - uses: octokit/request-action@v2.x
+        name: 'Open Code Freeze Milestone'
+        id: milestones
+        if: github.event_name == 'workflow_dispatch'
+        with:
+          route: PATCH /repos/{owner}/{repo}/milestones/115
```

Comment:
115 is hardcoded?

Reply:
Yup :D it's a specific milestone that we open and close to indicate when a code freeze is in place 🙂 Yes, it's very hacky 😂